tasks.py

```diff
@@ -19,14 +26,31 @@ def remove_check_file(path):
     if os.path.isfile(path):
         os.remove(path)
 
+def run_as(os_name):
+    if os_name.startswith('linux') or os_name == 'darwin':
+        return 'sudo'
+    elif os_name.startswith('Win32'):
```

Comment:
This could just be -
```python
else:
    return None
```
Or just stop at the if condition.

Reply:
Done